source/security/private-endpoints.txt

```diff
@@ -252,3 +366,35 @@ You can toggle this setting in three ways:
 - In the Admin API, call the :admin-api-endpoint:`Toggle Non-VPC Requests
   <operation/allNonVpcClientRequests>` with a boolean value (``true`` to remove
   the default restriction, or ``false`` to reinstate it.)
+
+Supported SDK Versions
+----------------------
+
+The Atlas Device SDKs can access your General and Sync endpoints. However, they
+must use a :ref:`version that supports the new domain 
+<domain-migration-sdk-versions>`.
+
+Private Endpoint Limitations
+----------------------------
+
+- You can't use Private Endpoints with globally deployed Apps or local
+  Apps deployed to Azure or GCP.
+
+- You can't use public internet connections to access your App if you
+  have a Private Endpoint enabled. All requests must come through the
+  Private Endpoint.
+
+- If you :ref:`change your deployment model <change-deployment-models>`,
+  then your existing Private Endpoints will not continue to work. You
+  will have to create new Private Endpoints for the App's new region.
```

Comment:
Nit: our style guide calls for us to avoid future tense. Maybe we can simplify this with something like: 

```suggestion
  then you must create new Private Endpoints for the App's new region.
```